extensions/adobe/experience/offer-management/personalized-offer.schema.json

```diff
@@ -0,0 +1,109 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/experience/offer-management/personalized-offer",
+  "meta:extends": ["https://ns.adobe.com/experience/offer-management/offer"],
+  "type": "object",
+  "title": "Adobe Experience Cloud Personalized Offer",
+  "description": "Adobe Experience Cloud Personalized Offer",
```

Comment:
All descriptions need to have less branding and more substance.

Reply:
fixed